k-distribution/include/builtin/domains.k

```diff
@@ -977,9 +979,22 @@ module K-IO
 // System
 
   syntax KItem ::= #system ( String ) [function, hook(IO.system), impure]
+                   // executes the given command line with `sh -c cmd:String` on unix or
+                   // `cmd /c cmd:String` on windows and returns #systemResult
+                 | #spawn  ( String ) [function, hook(IO.sharpSpawn), impure]
```

Comment:
See above comment. If you're going to make an implementation that invokes sh -c, that implementation should be hooked to IO.system and you should not have this function at all.